README.md

```diff
@@ -1,4 +1,681 @@
+	}
+
+	fmt.Printf("\tname: %s created\n", *createTopicOptionsModel.Name) 
+```
+
+### Deleting a Kafka topic
+
+To delete a Kafka topic, the admin REST SDK issues a DELETE request to the `/admin/topics/TOPICNAME`
+path (where `TOPICNAME` is the name of the Kafka topic that you want to delete).
+
+Expected return codes:
+- 202: Topic deletion request was accepted.
+- 403: Not authorized to delete topic.
+- 404: Topic does not exist.
+  
+A 202 (Accepted) status code is returned if the REST API accepts the delete
+request or status code 422 (Un-processable Entity) if the delete request is
+rejected. If a delete request is rejected then the body of the HTTP response
+will contain a [JSON object](#information-returned-when-a-request-fails) which
+provides additional information about why the request was rejected.
+
+Kafka deletes topics asynchronously. Deleted topics may still appear in the
+response to a [list topics request](#listing-kafka-topics) for a short period
+of time after the completion of a REST request to delete the topic.
+
+#### Example
+
+First create a delete topic options model which gives details of the topic.
+Then call the DeleteTopic function on the ibmEventStreamsAdminRestApiService service.
+
+```golang
+	// Construct an instance of the DeleteTopicOptions model
+	deleteTopicOptionsModel := new(ibmeventstreamsadminrestapiv1.DeleteTopicOptions)
+	deleteTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	deleteTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Delete Topic
+	response, operationErr := serviceAPI.DeleteTopic(deleteTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Deleting Topic: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Deleting Topic: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s deleted\n", *deleteTopicOptionsModel.TopicName)
+```
+
+### Listing Kafka topics
+
+You can list all of your Kafka topics by issuing a GET request to the
+`/admin/topics` path. 
+
+Expected status codes:
+  - 200: the topic list is returned as JSON in the following format:
+```json
+[
+  {
+    "name": "topic1",
+    "partitions": 1,
+    "retentionMs": 86400000,
+    "cleanupPolicy": "delete"
+  },
+  { "name": "topic2",
+    "partitions": 2,
+    "retentionMs": 86400000,
+    "cleanupPolicy": "delete"
+  }
+]
+```
+
+A successful response will have HTTP status code 200 (OK) and contain an
+array of JSON objects, where each object represents a Kafka topic and has the
+following properties:
+
+| Property name     | Description                                             |
+|-------------------|---------------------------------------------------------|
+| name              | The name of the Kafka topic.                            |
+| partitions        | The number of partitions assigned to the Kafka topic.   |
```

Comment:
`The number of partitions assigned to the Kafka topic.` -> `The number of partitions of the Kafka topic.`

Reply:
done